### ISSUE TITLE
complex enum in parameter descriptions

### ISSUE THREAD

**Issue by gr2m** [octokit/routes#6]
This one is challenging: https://developer.github.com/v3/projects/cards/#move-a-project-card, it creates

```
"options": [
  "top",
  "bottom",
  "after:<card_id>",
  "<card_id>",
  "id",
  "column_id"
],
```

with the current implementation.

**Comment by gr2m** [comment]
https://developer.github.com/v3/repos/#edit

```
"options": [
  "true",
  "false",
  "false",
  "422",
  "422"
],
```

**Comment by gr2m** [comment]
- https://developer.github.com/v3/issues/  many parameters marked as `string` should be `enum` instead. Same for 
	- https://developer.github.com/v3/issues/#list-issues,
	- https://developer.github.com/v3/issues/#list-issues-for-a-repository
	- https://developer.github.com/v3/issues/milestones/#list-milestones-for-a-repository
	- https://developer.github.com/v3/projects/#update-a-project
	- https://developer.github.com/v3/pulls/#list-pull-requests
	- https://developer.github.com/v3/pulls/comments/#list-comments-in-a-repository
	- https://developer.github.com/v3/repos/forks/#list-forks
	- https://developer.github.com/v3/search/#search-issues
- => `state`, `sort`, `direction`
- wrong enum values
	- https://developer.github.com/v3/projects/cards/#move-a-project-card
	- https://developer.github.com/v3/projects/columns/#move-a-project-column
- missing enum values
	- `merge_method`: https://developer.github.com/v3/pulls/#get-if-a-pull-request-has-been-merged
	- `event`: https://developer.github.com/v3/pulls/reviews/#submit-a-pull-request-review
	- `permissions`: https://developer.github.com/v3/repos/invitations/#update-a-repository-invitation

**Comment by gr2m** [comment]
this is from auditing the existing `@octokit/rest/lib/routes.json` file in https://github.com/octokit/rest.js/pull/822

- getStarredRepos
- getStarredReposForUser
- issues.edit (PATCH /repos/:owner/:repo/issues/:number)
- issues.getAll (GET /issues)
- issues.getForOrg (GET /orgs/:org/issues)
- issues.getForRepo (GET /orgs/:org/issues)
- issues.getForUser (GET /user/issues)
- getMilestones
- updateMilestone
- renderMarkdown: `mode`
- createTeam, editTeam: `privacy`
- org.update: `default_repository_permission`
- `getOrgProjects`: `state`
- `getRepoProjects`: `state`
- `updateProject`: `state`
- `createReview`: `event`
- `pullRequests.getAll`: `direction`, `sort`
- `getCommentsForRepo`: `direction`, `sort`
- `merge`: `merge_method`
- `submitReview`: `event`
- `pullRequests.update`: `state`
- `reactions.createForCommitComment`, `createForIssue`, `createForIssueComment`, `createForPullRequestReviewComment`, `getForCommitComment`, `getForIssue`, `getForIssueComment`, `getForPullRequestReviewComment`: `content`
- getArchiveLink: archive_format
- getForks: `sort`
- updateInvite: permissions
- search.code, search.commits, search.issues, search.repos, search.users: order, sort
- editOrgMembership, getOrgMemberships: `state`

**Comment by gr2m** [comment]
closed by #91
